### ISSUE TITLE
IBT: Leads 

### ISSUE THREAD

**Issue by madcharlie02** [hackforla/incubator#22]
### Overview
This issue tracks the agenda for our weekly meetings

#### Issue Template
```
## [Date ]  Meeting Agenda

### Prework to prep for meeting

### Recurring items
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/incubator/projects/1#column-19648741)
 - [ ] reviewing any issues in the [questions and review column](https://github.com/hackforla/incubator/projects/1#column-19648744)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Charles](https://github.com/hackforla/incubator/issues/assigned/madcharlie02)
       - [ ] [Bonnie](https://github.com/hackforla/incubator/issues/assigned/ExperimentsInHonesty)
       - [ ] [Jason](https://github.com/hackforla/incubator/issues/assigned/JasonEb)
       - [ ] [Judson](https://github.com/hackforla/incubator/issues/assigned/nyarly)
       - [ ] [Robin](https://github.com/hackforla/incubator/issues/assigned/robinglov)
       - [ ] [Jenny](https://github.com/hackforla/incubator/issues/assigned/jennyc910)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label](https://github.com/hackforla/incubator/issues?q=is%3Aopen+is%3Aissue+label%3Aepic) and define how to break down further
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  hackforla/ops#27 hackforla/ops#47

### New Items
- [ ] 
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting

### Task Items

```

### Links to other agenda issues for your team 
(so that you can easily post on their agendas items that come up at yours)

**Comment by madcharlie02** [comment]
## 2023-10-03  Meeting Agenda

### Prework to prep for meeting
- [ ] #

### Recurring items: Happens on the ________ meeting
 - [ ] review any issues that are in the [new issue approval column]
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Name of Person]
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label] and define how to break down further
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### New Items
- [ ] 
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
- How many members are in each project, and their tasks
- 

### Notes from Meeting

### Task Items

- [ ] Disect open issues and their updates

**Comment by madcharlie02** [comment]
## 2023-11-14  Meeting Agenda

### Prework to prep for meeting
- [ ] #24 

### Recurring items
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/incubator/projects/1#column-19648741)
 - [ ] reviewing any issues in the [questions and review column](https://github.com/hackforla/incubator/projects/1#column-19648744)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Charles](https://github.com/hackforla/incubator/issues/assigned/madcharlie02)
       - [ ] [Bonnie](https://github.com/hackforla/incubator/issues/assigned/ExperimentsInHonesty)
       - [ ] [Jason](https://github.com/hackforla/incubator/issues/assigned/JasonEb)
       - [ ] [Judson](https://github.com/hackforla/incubator/issues/assigned/nyarly)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label](https://github.com/hackforla/incubator/issues?q=is%3Aopen+is%3Aissue+label%3Aepic) and define how to break down further
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### New Items
- [ ] 
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting

### Task Items
- Make an onboarding and offboarding issue

**Comment by madcharlie02** [comment]
## 2023-11-28  Meeting Agenda

### Prework to prep for meeting
- [ ] #24 

### Recurring items
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/incubator/projects/1#column-19648741)
 - [ ] reviewing any issues in the [questions and review column](https://github.com/hackforla/incubator/projects/1#column-19648744)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Charles](https://github.com/hackforla/incubator/issues/assigned/madcharlie02)
       - [ ] [Bonnie](https://github.com/hackforla/incubator/issues/assigned/ExperimentsInHonesty)
       - [ ] [Jason](https://github.com/hackforla/incubator/issues/assigned/JasonEb)
       - [ ] [Judson](https://github.com/hackforla/incubator/issues/assigned/nyarly)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label](https://github.com/hackforla/incubator/issues?q=is%3Aopen+is%3Aissue+label%3Aepic) and define how to break down further
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### New Items
- [ ] 
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting

### Task Items
- Make an onboarding and offboarding issue

**Comment by robinglov** [comment]
## [Date ]  Meeting Agenda

### Prework to prep for meeting
- [ ] #24 

### Recurring items
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/incubator/projects/1#column-19648741)
 - [ ] reviewing any issues in the [questions and review column](https://github.com/hackforla/incubator/projects/1#column-19648744)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Charles](https://github.com/hackforla/incubator/issues/assigned/madcharlie02)
       - [ ] [Bonnie](https://github.com/hackforla/incubator/issues/assigned/ExperimentsInHonesty)
       - [ ] [Jason](https://github.com/hackforla/incubator/issues/assigned/JasonEb)
       - [ ] [Judson](https://github.com/hackforla/incubator/issues/assigned/nyarly)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label](https://github.com/hackforla/incubator/issues?q=is%3Aopen+is%3Aissue+label%3Aepic) and define how to break down further
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### New Items
- [ ] 
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting

### Task Items
- Make an onboarding and offboarding issue

**Comment by robinglov** [comment]
## [2024-01-16]  Meeting Agenda

### Prework to prep for meeting
- [ ] #24 

### Recurring items
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/incubator/projects/1#column-19648741)
 - [ ] reviewing any issues in the [questions and review column](https://github.com/hackforla/incubator/projects/1#column-19648744)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Charles](https://github.com/hackforla/incubator/issues/assigned/madcharlie02)
       - [ ] [Bonnie](https://github.com/hackforla/incubator/issues/assigned/ExperimentsInHonesty)
       - [ ] [Jason](https://github.com/hackforla/incubator/issues/assigned/JasonEb)
       - [ ] [Judson](https://github.com/hackforla/incubator/issues/assigned/nyarly)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label](https://github.com/hackforla/incubator/issues?q=is%3Aopen+is%3Aissue+label%3Aepic) and define how to break down further
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### New Items
- [ ] Review issue [#37](https://github.com/hackforla/incubator/issues/37) and [#36](https://github.com/hackforla/incubator/issues/36) for clarification. #37 seems to be dependent on #36 but it doesn't show a dependency.
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting

### Task Items
- Make an onboarding and offboarding issue

**Comment by robinglov** [comment]
## 2024-1-2024  Meeting Agenda

### Prework to prep for meeting
- [ ] #24 

### Recurring items
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/incubator/projects/1#column-19648741)
 - [ ] reviewing any issues in the [questions and review column](https://github.com/hackforla/incubator/projects/1#column-19648744)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Charles](https://github.com/hackforla/incubator/issues/assigned/madcharlie02)
       - [ ] [Bonnie](https://github.com/hackforla/incubator/issues/assigned/ExperimentsInHonesty)
       - [ ] [Jason](https://github.com/hackforla/incubator/issues/assigned/JasonEb)
       - [ ] [Judson](https://github.com/hackforla/incubator/issues/assigned/nyarly)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label](https://github.com/hackforla/incubator/issues?q=is%3Aopen+is%3Aissue+label%3Aepic) and define how to break down further
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### New Items
- [ ] 
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting

### Task Items

**Comment by nyarly** [comment]
@robinglov "The incubator will need to be dockerized."

There are two misconceptions here. I'm not 100% clear where they arise, so forgive me from covering all the bases, which for sure means I'll be saying things you already know.

First, Incubator is a) the AWS project where we host HfLA projects before they're ready for their own deployments, and b) this whole Git repo, where the Terraform code to manage that project lives. New HfLA volunteers won't be creating incubators, but adding projects to Incubator.

Second, it's won't be dockerized. "Dockerizing" is the process of taking an application that runs conventionally - e.g. a developer executes it locally, or sets it up to run directly on the host system, and creating a `Dockerfile` for it in order to build and publish a Docker image. Incubator code doesn't run that way, so dockerizing it isn't sensible. I wouldn't expect the Incubator project to **ever** include a Dockerfile. 

_In general_ no one in the Ops CoP should be dockerizing anything as part of their Ops responsibilities. Each project's developers knows their application better than Ops could, and should be responsible for writing their Dockerfile and publishing the docker image. Ops will provide them with a repo name to publish to. That said, because that's the handoff between Dev and Ops (and having a clear handoff that's the same for every project is hugely valuable), Ops should understand that process, so it makes sense for there to be some training about how to create a Docker image - so that Ops engineers can help train Dev in doing it.

Does that all make sense? I know it's a lot to come out of one sentence, but I think it's important to all be aligned early in the project. Please feel free to @ me in Slack, or schedule some time to go over it.

**Comment by robinglov** [comment]
> Does that all make sense?

Thank you @nyarly . This will be immensely helpful to have us all aligned in future discussions.

**Comment by chelseybeck** [comment]
@robinglov I created a [Google Doc](https://docs.google.com/document/d/1OUwPCzzHa2YROSp7YugYG_LH-NNIiroweqVxxSrQ4gY/edit?usp=sharing) to store this stuff and track versions

**Comment by robinglov** [comment]
## [Date ]  Meeting Agenda

### Prework to prep for meeting
- [ ] #24 

### Recurring items
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/incubator/projects/1#column-19648741)
 - [ ] reviewing any issues in the [questions and review column](https://github.com/hackforla/incubator/projects/1#column-19648744)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Charles](https://github.com/hackforla/incubator/issues/assigned/madcharlie02)
       - [ ] [Bonnie](https://github.com/hackforla/incubator/issues/assigned/ExperimentsInHonesty)

       - [ ] [Judson](https://github.com/hackforla/incubator/issues/assigned/nyarly)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label](https://github.com/hackforla/incubator/issues?q=is%3Aopen+is%3Aissue+label%3Aepic) and define how to break down further
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #

### New Items
- [ ] Incubator PR process
- [ ] Onboarding flow
- [ ] IAM read-only access


### FYIs
- #28 #29 #46 PRs are ready for review

### Notes from Meeting

### Task Items
- [ ] Carry over Items to next meeting that don't get addressed

**Comment by madcharlie02** [comment]
**Template to Use**

# Data Engineer (DevOps)

This team member will be responsible for developing data pipelines to clean, transform, and process data. They will ensure data integration between systems, including deployment of machine-learning pipelines where necessary.
 
## Responsibilities
- Create and maintain optimal data pipeline architecture.
- Assemble large, complex data sets that meet functional / non-functional business requirements.
- Identify, design, and implement internal process improvements: automating manual processes, optimizing data delivery, re-designing infrastructure for greater scalability, etc.
- Build the infrastructure required for optimal extraction, transformation, and loading of data from a wide variety of data sources using SQL and AWS ‘big data’ technologies.
- Build analytics tools that utilize the data pipeline to provide actionable insights into customer acquisition, operational efficiency and other key business performance metrics.
- Work with stakeholders including the Executive, Product, Data and Design teams to assist with data-related technical issues and support their data infrastructure needs.
- Create data tools for analytics and data scientist team members that assist them in building and optimizing our product into an innovative industry leader.

## Basic Qualifications
- Working SQL knowledge (or interest)  and experience working with relational databases, query authoring (SQL) as well as working familiarity with a variety of databases.
- Experience or interest building and optimizing ‘big data’ data pipelines, architectures and data sets.
- Experience or interest performing root cause analysis on internal and external data and processes to answer specific business questions and identify opportunities for improvement.
- Strong analytical skills related to working with unstructured datasets.
- Build processes supporting data transformation, data structures, metadata, dependency and workload management.
- An interest in manipulating, processing and extracting value from large disconnected datasets.
- Strong project management and organizational skills.
- Experience or interest in/with object-oriented/object function scripting languages: Python, Java, C++, Scala, etc.

**Comment by madcharlie02** [comment]
**Draft for Current Open Role**

# Developer (Terraform)

This team member will develop data pipelines to clean, transform, and process data. They will ensure data integration between systems, including code deployment within Terraform, an Amazon Webservice platform, where necessary. They will design and implement effective database solutions and models to store and retrieve company data. Examine and identify database structural necessities by evaluating client operations, applications, and programming.
 
## Responsibilities
- Create and maintain optimal data pipeline architecture.
- Assemble large, complex data sets that meet functional / non-functional business requirements.
- Identify, design, and implement internal process improvements: automating manual processes, optimizing data delivery, re-designing infrastructure for greater scalability, etc.
- Build the infrastructure required for optimal extraction, transformation, and loading of data from a wide variety of data sources using SQL and AWS ‘big data’ technologies.
- Build analytics tools that utilize the data pipeline to provide actionable insights into customer acquisition, operational efficiency and other key business performance metrics.
- Work with stakeholders including the Executive, Product, Data and Design teams to assist with data-related technical issues and support their data infrastructure needs.

## Basic Qualifications
- Working python knowledge (or interest)  and experience working with relational databases
- Working Terraform knowledge (or interest)  and experience working with relational databases
- Experience or interest building and optimizing ‘big data’ data pipelines, architectures and data sets.
- Experience or interest performing root cause analysis on internal and external data and processes to answer specific business questions and identify opportunities for improvement.
- Strong analytical skills related to working with unstructured datasets.
- Build processes supporting data transformation, data structures, metadata, dependency and workload management.
- An interest in manipulating, processing and extracting value from large disconnected datasets.
- Strong project management and organizational skills.
- Experience or interest in/with object-oriented/object function scripting languages: Python, Java, C++, Scala, etc.

**Comment by robinglov** [comment]
## 2024-02-6  Meeting Agenda

### Prework to prep for meeting
- [ ] #24 

### Recurring items
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/incubator/projects/1#column-19648741)
 - [ ] reviewing any issues in the [questions and review column](https://github.com/hackforla/incubator/projects/1#column-19648744)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Charles](https://github.com/hackforla/incubator/issues/assigned/madcharlie02)
       - [ ] [Bonnie](https://github.com/hackforla/incubator/issues/assigned/ExperimentsInHonesty)
       - [ ] [Jason](https://github.com/hackforla/incubator/issues/assigned/JasonEb)
       - [ ] [Judson](https://github.com/hackforla/incubator/issues/assigned/nyarly)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label](https://github.com/hackforla/incubator/issues?q=is%3Aopen+is%3Aissue+label%3Aepic) and define how to break down further
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #27

### New Items
- [ ] #52 
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting

### Task Items
- Make an onboarding and offboarding issue

**Comment by robinglov** [comment]
## [2024-02-13 ]  Meeting Agenda

### Prework to prep for meeting
- [ ] #24 

### Recurring items
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/incubator/projects/1#column-19648741)
 - [ ] reviewing any issues in the [questions and review column](https://github.com/hackforla/incubator/projects/1#column-19648744)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label](https://github.com/hackforla/incubator/issues?q=is%3Aopen+is%3Aissue+label%3Aepic) and define how to break down further
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #27 #47

### New Items
- [ ] 
- [ ] Carry over Items to next meeting that don't get addressed

### FYIs
-

### Notes from Meeting

### Task Items

**Comment by ExperimentsInHonesty** [comment]
## 2024-02-19 Meeting Agenda

### Prework to prep for meeting
- [ ] #24 

### Recurring items
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/incubator/projects/1#column-19648741)
 - [ ] reviewing any issues in the [questions and review column](https://github.com/hackforla/incubator/projects/1#column-19648744)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Charles](https://github.com/hackforla/incubator/issues/assigned/madcharlie02)
       - [ ] [Bonnie](https://github.com/hackforla/incubator/issues/assigned/ExperimentsInHonesty)
       - [ ] [Jason](https://github.com/hackforla/incubator/issues/assigned/JasonEb)
       - [ ] [Judson](https://github.com/hackforla/incubator/issues/assigned/nyarly)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label](https://github.com/hackforla/incubator/issues?q=is%3Aopen+is%3Aissue+label%3Aepic) and define how to break down further
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #27 #47

### New Items
- [ ] BW: Are we putting this on hold until we get the levels set up, or can we proceed? ## 

### FYIs
-

### Notes from Meeting

### Task Items

**Comment by madcharlie02** [comment]
## 2024-02-27 Meeting Agenda

### Prework to prep for meeting
- [ ] #24 

### Recurring items
 - [ ] review any issues that are in the [new issue approval column](https://github.com/hackforla/incubator/projects/1#column-19648741)
 - [ ] reviewing any issues in the [questions and review column](https://github.com/hackforla/incubator/projects/1#column-19648744)
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each Team member
       - [ ] [Charles](https://github.com/hackforla/incubator/issues/assigned/madcharlie02)
       - [ ] [Bonnie](https://github.com/hackforla/incubator/issues/assigned/ExperimentsInHonesty)
       - [ ] [Jason](https://github.com/hackforla/incubator/issues/assigned/JasonEb)
       - [ ] [Judson](https://github.com/hackforla/incubator/issues/assigned/nyarly)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label](https://github.com/hackforla/incubator/issues?q=is%3Aopen+is%3Aissue+label%3Aepic) and define how to break down further
- [ ] Review to see if there are any members to Onboard or Offboard 
      - [ ]  #27 #47

### New Items
- [ ] BW: Are we putting this on hold until we get the levels set up, or can we proceed?  #45 

### FYIs
-

### Notes from Meeting

### Task Items

**Comment by ExperimentsInHonesty** [comment]
Closing this issue because we no longer have Incubator as a separate team.
